modules/rest-api/pages/rest-retrieve-cluster-rebalance-reason-codes.adoc

```diff
@@ -0,0 +1,97 @@
+= Retrieving Cluster Rebalance Reason Codes
+
+:description: pass:q[If the cluster requires rebalancing, then the reason(s) for the rebalance can be ascertained using the `GET /pools/default` method.]
```

Comment:
When the cluster needs to be rebalanced, you can retrieve the reason for the rebalance using the 'GET /pools/default' method.